### PR TITLE
BOM-1044 Django22 Support

### PR DESCRIPTION
**Pull request by Ayub-Khan**
-updated tox and Travis config
-fixed django22 issues
-added trove classifiers

### CODE REVIEW

**Comment by Ayub-Khan** [comment]
@jmbowman kindly review.

File: .travis.yml [hunk 1]
```diff
@@ -3,26 +3,27 @@ language: python
...
   exclude:
     - python: 2.7
       env: TOXENV=django20
     - python: 2.7
       env: TOXENV=django21
+    - python: 2.7
+      env: TOXENV=django22
   allow_failures:
-    - python: 3.6
+    - python: 3.5
```
Comment (awais786) [inline .travis.yml]: why allowing ?

Comment (awais786) [reply .travis.yml]: I think you should remove it.

File: workbench/settings.py [hunk 2]
```diff
@@ -135,7 +135,7 @@
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = '5ftdd9(@p)tg&amp;bqv$(^d!63psz9+g+_i5om_e%!32%po2_+%l7'
 
-MIDDLEWARE_CLASSES = [
+MIDDLEWARE_CLASSES = MIDDLEWARE = [
```
Comment (awais786) [inline workbench/settings.py]: please remove `MIDDLEWARE_CLASSES =`

File: pylintrc_tweaks [hunk 3]
```diff
@@ -2,3 +2,4 @@
 [MASTER]
 ignore = migrations
 load-plugins = edx_lint.pylint,pylint_django
+disable = model-missing-unicode
```
Comment (jmbowman) [inline pylintrc_tweaks]: Do you actually need this?  It looks like you fixed the one spot likely to trigger this warning.

File: workbench/urls.py [hunk 4]
```diff
@@ -61,7 +61,7 @@
         name='reset_state'
     ),
 
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^admin/', admin.site.urls),
```
Comment (jmbowman) [inline workbench/urls.py]: Why was this change needed?  I didn't think `include()` had been deprecated.

Comment (Ayub-Khan) [reply workbench/urls.py]: it was causing some namespace issues in later versions of django22. As this was changed 1.9 onwards so removed it.

Comment (jmbowman) [reply workbench/urls.py]: Ah, I'd missed/forgotten that change specifically for Django admin URLs.
